android/app/src/main/kotlin/com/linagora/flutter/linshare_flutter_app/MainActivity.kt

```diff
@@ -1,6 +1,11 @@
 package com.linagora.flutter.linshare_flutter_app
 
-import io.flutter.embedding.android.FlutterActivity
+import io.flutter.embedding.android.FlutterFragmentActivity
+import io.flutter.embedding.engine.FlutterEngine
+import io.flutter.plugins.GeneratedPluginRegistrant
 
-class MainActivity: FlutterActivity() {
+class MainActivity: FlutterFragmentActivity() {
+    override fun configureFlutterEngine(flutterEngine: FlutterEngine) {
+        GeneratedPluginRegistrant.registerWith(flutterEngine)
+    }
```

Comment:
Can you say sth about that?

Reply:
To use flutter's local_auth plugin we need to use a FragmentActivity instead of Activity. So here we configure the MainActivity class to integrate that plugin.